sdk-springboot/src/main/java/io/dapr/springboot/DaprApplication.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.Properties;
+
+/**
+ * Dapr's callback implementation via SpringBoot.
+ */
+@SpringBootApplication
+public class DaprApplication {
+
+  /**
+   * Starts Dapr's callback in a given port.
+   * @param port Port to listen to.
+   */
+  public static void start(int port) {
+    SpringApplication app = new SpringApplication(DaprApplication.class);
+    Properties properties = new Properties();
+    properties.setProperty("server.port", Integer.toString(port));
+    app.setDefaultProperties(properties);
+    app.run();

```

Comment:
You should be able to simply call `main("--server.port=" +  port)` instead of these 5 lines of code.

Reply:
Will do.